projects/zlib-ng/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google Inc.
+# Copyright 2020 Google Inc.
```

Comment:
please revert changes in this file, seems accidental

Reply:
Oops, should be fixed